src/modules/InviteUser/components/invitationActions.vue

```diff
@@ -47,7 +66,35 @@ export default {
               //error message in a dialog box
             }
         },
-    },
+        async deleteMember(){
+          try{
+            // const response = await deleteUser(this.id);
+            // if (response.data.status === true){
+              const response = await axios.delete(
+          `https://6319958e8e51a64d2be7568b.mockapi.io/invitedUsers/${this.id}`   // used only for testing
+        );
+              this.$emit('deleteInvite')
+            // }
+            //show success message
```

Comment:
Show a success message with "alert()" at least.

Reply:
pr for toast message is being created immediately after this